client/my-sites/customer-home/main.jsx

```diff
@@ -66,6 +68,17 @@ const Home = ( { canUserUseCustomerHome, site, siteId, trackViewSiteAction, noti
 				align="left"
 				hasScreenOptions
 			/>
+
+			<div className="customer-home__site-content">
+				<SiteIcon site={ site } size={ 58 } />
+				<div className="customer-home__site-info">
+					<div className="customer-home__site-title">{ site.name }</div>
+					<ExternalLink href={ site.URL } className="customer-home__site-domain">
```

Comment:
Personally, I don't preemptively add classes that aren't needed yet. Other folks might do differently though.

Reply:
We should also retain the `calypso_customer_home_my_site_view_site_click` tracks event since this link is equivalent to the Visit Site button. Should just be a matter of an `onClick` handler:

```suggestion
					<ExternalLink href={ site.URL } className="customer-home__site-domain" onClick={ trackViewSiteAction }>
```

Reply:
Thanks for catching this.